src/dotnet/commands/dotnet-new/Program.cs

```diff
@@ -88,7 +88,7 @@ public static int Run(string[] args)
             var dotnetNew = new NewCommand();
             app.OnExecute(() => {
 
-                var csharp = new { Name = "C#", Alias = new[] { "c#", "cs", "csharp" }, TemplatePrefix = "CSharp", Templates = new[] { "Console" } };
+                var csharp = new { Name = "C#", Alias = new[] { "c#", "cs", "csharp" }, TemplatePrefix = "CSharp", Templates = new[] { "Console", "Lib", "xunittest" } };
```

Comment:
This should be tracked as an issue. It's silly we're defining the strings here and yet also have them in template names.